### PR TITLE
Bundle size locks for every entrypoint

### PR DESCRIPTION
**Pull request by dzearing**
This change adds bundle size locks not just for the button bundle, but each individual one.

This will help understand the impact of the changes and preemptively catch bad import problems which pull more content than they expected.

This will also help us understand the impact that tree shaking has!

### CODE REVIEW

File: apps/test-bundles/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+const path = require('path');
+const resources = require('../../scripts/tasks/webpack-resources');
+
+const AllEntries = [
```
Comment (dzearing) [inline apps/test-bundles/webpack.config.js]: I am not sure how to get this list in the best way.

We could put an "entries" section in the `office-ui-fabric-react/package.json` file and read that.

File: apps/test-bundle-button/webpack.config.js [hunk 2]
```diff
@@ -3,13 +3,15 @@ const resources = require('../../scripts/tasks/webpack-resources');
 
 const PACKAGE_NAME = 'test-bundle-button';
 
+const entry = {
```
Comment (dzearing) [inline apps/test-bundle-button/webpack.config.js]: This package probably should be removed.

File: .vscode/launch.json [hunk 3]
```diff
@@ -52,7 +52,7 @@
         "--",
         "--production"
       ],
-      "cwd": "${workspaceRoot}/apps/fabric-website",
+      "cwd": "${workspaceRoot}/apps/test-bundles",
```
Comment (dzearing) [inline .vscode/launch.json]: probably not needed.

File: scripts/package.json [hunk 4]
```diff
@@ -49,8 +49,224 @@
   },
   "bundlesize": [
     {
-      "path": "../apps/test-bundle-button/dist/test-bundle-button.min.js",
-      "maxSize": "47.6 kB"
+      "path": "../apps/test-bundles/dist/ActivityItem.min.js",
+      "maxSize": "30.0 kB"
```
Comment (dzearing) [inline scripts/package.json]: these sizes will be updated.

File: scripts/tasks/webpack.js [hunk 5]
```diff
@@ -78,5 +78,6 @@ function getFileSize(size) {
   } else {
     sizeString = (Math.round(1000 * size / 1024) / 1000) + ' KB'
   }
-  return chalk.cyan(sizeString);
+  // return chalk.cyan(sizeString);
```
Comment (dzearing) [inline scripts/tasks/webpack.js]: This function is unneeded.

File: apps/test-bundles/package.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "test-bundles",
+  "version": "5.0.1",
```
Comment (cliffkoh) [inline apps/test-bundles/package.json]: What's the logic behind this version? Does it track OUFR's major version?

File: apps/test-bundles/webpack.config.js (hunk 1 above)
Comment (cliffkoh) [reply apps/test-bundles/webpack.config.js]: Dynamically generate by reading the files at the top level of OUFR that looks like a top level import? (i.e. all pascal cased files?)

**Comment by dzearing** [comment]
I'm going to close this. The `bundlesize` plugin will not handle this. We need to investigate alternative solutions.

**Pull request closed without merging**